scripts/py_matter_idl/matter_idl/test_generators.py

```diff
@@ -122,6 +122,10 @@ def _create_generator(self, storage: GeneratorStorage, idl: Idl):
             return BridgeGenerator(storage, idl)
         if self.generator_name.lower() == 'cpp-app':
             return CppApplicationGenerator(storage, idl)
+        if self.generator_name.lower() == 'custom':
+            sys.path.append('../examples')
```

Comment:
This assumes a location from which this script is run. I do not think this can be assumed.

Generally within pigweed you should setup package dependencies to have imports "just work" and otherwise you probably want absolute paths here.

Reply:
made path absolute